### ISSUE TITLE
Question: Is a new/xmlrpc going to be released ever @ PECL providing PHP 8.1 compatibility?

### ISSUE THREAD

**Issue by stronk7** [php/pecl-networking-xmlrpc#2]
Hi,

few minutes ago I saw #1 , and I'm well aware about the current abandoned / not maintained status of https://pecl.php.net/package/xmlrpc (since 8.0).

Just for the sake of being 100% sure... is the solution applied @ #1 going to be included in a new "RC3" (or so) release, providing 8.1 compatibility?

Asking because we still build our PHP images with xmlrpc because of some (extra, not-critical) functionalities and, while we are in the process of looking for alternatives (userland, abandon protocol...) we still have a good number of tests and stuff needing it.

Ciao :-)

**Comment by cmb69** [comment]
I have just released xmlrpc-1.0.0RC3.

**Comment by stronk7** [comment]
Yay, heap thanks!

**Comment by gggeek** [comment]
@stronk7 in case you have not yet migrated away from the pecl library, please be aware that there exists a pure-php reimplementation of the same API, which might be useful in easing migration scenarios: phpxmlrpc/polyfill-xmlrpc

**Comment by stronk7** [comment]
Thanks, @gggeek .

Yeah, [we have recently moved](https://tracker.moodle.org/browse/MDL-76055) all remaining uses from the extension to your nice library (not using the polyfill, though I borrowed a few tricks from it, thanks!). And one plugin (xmlrpc webservices support) that is [still using the pecl extension has been removed](https://tracker.moodle.org/browse/MDL-76052) from core.

So, basically, we are now 100% liberated from the PECL extension, yay!
